cardano-testnet/src/Testnet/Cardano.hs

```diff
@@ -470,20 +471,45 @@ testnet testnetOptions H.Conf {..} = do
 
   -- Make the pool operator cold keys
   -- This was done already for the BFT nodes as part of the genesis creation
-  forM_ poolNodeNames $ \node -> do
+
+        -- poolVrfVkeys
```

Comment:
Can you elaborate on this more? How was it done?

Reply:
How what was done?  This was testnet was a direct port of the `./scripts/byron-to-alonzo/mkfiles.sh` at the time it was ported.

Reply:
Ah I now understand the comment. The pool operator keys were created during genesis creation.